### PR TITLE
add `error_set` rendering

### PR DESCRIPTION
**Pull request by Rexicon226**

### CODE REVIEW

File: lib/index.html [hunk 1]
```diff
@@ -315,7 +315,7 @@ <h2>Errors</h2>
       <div id="fnErrorsAnyError">
         <p><span class="tok-type">anyerror</span> means the error set is known only at runtime.</p>
       </div>
-      <div id="tableFnErrors"><dl id="listFnErrors"></dl></div>
+      <div id="tableFnErrors"><ul id="listFnErrors"></ul></div>
```
Comment (andrewrk) [inline lib/index.html]: can I see a pic? old vs new view

**Review by andrewrk — CHANGES_REQUESTED** [review]

File: lib/index.html (hunk 1 above)
Comment (Rexicon226) [reply lib/index.html]: Before:
image

After:
image

Comment (andrewrk) [reply lib/index.html]: I think it looks better with the dl, dt, dd

Comment (Rexicon226) [reply lib/index.html]: Sure, I'll switch it back to `dl`.

**Pull request closed without merging**